frontend/src/component/feature/StrategyTypes/FlexibleStrategy/StickinessSelect/StickinessSelect.tsx

```diff
@@ -1,14 +1,46 @@
-import Select from 'component/common/select';
-import { type SelectChangeEvent, useTheme } from '@mui/material';
+import {
+    MenuItem,
+    styled,
+    useTheme,
+    Select,
+    type SelectChangeEvent,
+    FormControl,
+    InputLabel,
+} from '@mui/material';
 import { useStickinessOptions } from 'hooks/useStickinessOptions';
+import { SELECT_ITEM_ID } from 'utils/testIds';
+import type { ReactNode } from 'react';
 
 interface IStickinessSelectProps {
     label: string;
     value: string | undefined;
     editable: boolean;
-    onChange: (event: SelectChangeEvent) => void;
+    onChange: (event: SelectChangeEvent<string>) => void;
     dataTestId?: string;
 }
+
+const StyledValueContainer = styled('div')(({ theme }) => ({
+    lineHeight: 1.1,
+    marginTop: -2,
+    marginBottom: -10,
+}));
+
+const StyledLabel = styled('div')(({ theme }) => ({
+    fontSize: theme.fontSizes.smallBody,
+}));
+
+const StyledDescription = styled('div')(({ theme }) => ({
```

Comment:
Small thing, but this should probably be a `p`. The label? Yea ... possibly. And you know, maybe they should be definition lists, but let's not go that far right now.

```suggestion
const StyledDescription = styled('p')(({ theme }) => ({
```